test/test_unit.jl

```diff
@@ -1633,6 +1639,39 @@ end
     @test mesh.boundary_faces[:entire_boundary] == [1, 2]
 end
 
+@testset "PERK Single p2 Constructors" begin
+    path_coeff_file = mktempdir()
+    Trixi.download("https://gist.githubusercontent.com/DanielDoehring/8db0808b6f80e59420c8632c0d8e2901/raw/39aacf3c737cd642636dd78592dbdfe4cb9499af/MonCoeffsS6p2.txt",
+                   joinpath(path_coeff_file, "gamma_6.txt"))
+
+    ode_algorithm = Trixi.PairedExplicitRK2(6, path_coeff_file)
+
+    @test isapprox(ode_algorithm.a_matrix,
+                   [0.12405417889682908 0.07594582110317093
+                    0.16178873711001726 0.13821126288998273
+                    0.16692313960864164 0.2330768603913584
+                    0.12281292901258256 0.37718707098741744], atol = 1e-13)
```

Comment:
Are these results deterministic even if the optimizers change in the future?

Reply:
I don't think so. However, the tolerance might have to be increased. @DanielDoehring

Reply:
Good question. In principle, the ECOS solver is settled and the optimization problem is convex, so I expect little variation in the results. However, as there are even differences when changing the Julia version, I would not take this for certain.

Reply:
Ok, let's see whether it becomes annoying in the future.